sqlmesh/core/model/meta.py

```diff
@@ -465,6 +520,67 @@ def custom_materialization_properties(self) -> CustomMaterializationProperties:
             return self.kind.materialization_properties
         return {}
 
+    @cached_property
+    def grants(self) -> t.Optional[GrantsConfig]:
```

Comment:
Seems a little weird to go through the trouble of converting `t.Dict[str, t.List[str]]` into `exp.Tuple` just to convert it back again here.